Makefile

```diff
@@ -182,3 +182,10 @@ appimage-%:
 	bash scripts/genappimage.sh $*
 
 .PHONY: test clean distclean nvim libnvim cmake deps install appimage checkprefix benchmark $(FORMAT) $(LINT) $(TEST)
+
+.PHONY: emmylua-check
+emmylua-check: luv
+	emmylua_check --config=$(PWD)/.luarc.json runtime/lua
```

Comment:
At least for the purpose of testing in this PR, I'd just vendor these files as `runtime/lua/vim/uv/_meta/*`; then emmyluals will pick them up automatically (also in downstream projects).

Reply:
Feel free to pull in the commit from https://github.com/neovim/neovim/pull/33375 in this PR.